generator/input/new_description.json

```diff
@@ -33,19 +33,37 @@
         {
           "name": "end1",
           "protocol": "http",
-          "cpu_consumption": 0.003,
-          "network_consumption": 0.002,
-          "memory_consumption": 0.003,
-          "forward_requests": "asynchronous",
-          "called_services": [
-            {
-              "service": "service2",
-              "port": "80",
-              "endpoint": "end2",
-              "protocol": "http",
-              "traffic_forward_ratio": 1
-            }
-          ]
+          "execution_mode": "sequential",
+          "cpu_complexity": {
+            "execution_time": "10s",
```

Comment:
As discussed before, we should reduce the execution_times in the scale of miliseconds per requests.

Reply:
It seems the minimum timeout supported by stress-ng is 1 second...

Reply:
Is it possible to kill the stress-ng subprocess in python? so we just pass a huge timeout to stress-ng and then kill it will python

Reply:
It could be but the load stress-ng generates takes some time to be applied and reflected on resource consumption. If we want to have minimum processing delay in the order of milliseconds we could instead simulate a sleep task or just don't configure any cpu or memory task...

Reply:
sleep task does not create any load. I'll check it out.

Reply:
yes, it doesn't create any load... I think it is hard to simulate tasks that are both cpu-bounded and I/O bounded at the same time... either we can simulate tasks that have a heavy load on CPU/memory or on network, but not both...

Reply:
After discussion, this will be solved in issue #48.